src/jvm/test/resources/expected_logs/run_concurrent_test/hashmap.txt

```diff
@@ -7,80 +7,82 @@ java.lang.IllegalStateException: Check failed.
+|                                                                                                                                                                                 |                                                                    |       putVal(0, 0, 1, false, true): null at HashMap.put(HashMap.java:610)                                                               |
+|                                                                                                                                                                                 |                                                                    |         table ➜ null at HashMap.putVal(HashMap.java:626)                                                                                |
+|                                                                                                                                                                                 |                                                                    |         resize(): Array<Node>#1 at HashMap.putVal(HashMap.java:627)                                                                     |
+|                                                                                                                                                                                 |                                                                    |         Array<Node>#1[0] ➜ null at HashMap.putVal(HashMap.java:628)                                                                     |
+|                                                                                                                                                                                 |                                                                    |         switch                                                                                                                          |
+|                                                                                                                                                                                 | run()                                                              |                                                                                                                                         |
+|                                                                                                                                                                                 |   Lambda#2.invoke() at ThreadsKt$thread$thread$1.run(Thread.kt:30) |                                                                                                                                         |
```

Comment:
Why `Lambda#2.invoke()` started to apper in the short trace? Can we hide it from the short trace?

Reply:
fixed